measure-builds/src/main/java/com/automattic/android/measure/reporters/InternalWooCommerceA8cCiReporter.kt

```diff
@@ -22,38 +20,38 @@ import io.ktor.http.HttpHeaders
 import io.ktor.http.HttpHeaders.Authorization
 import io.ktor.http.HttpStatusCode
 import io.ktor.http.contentType
-import kotlinx.serialization.encodeToString
-import kotlinx.serialization.json.Json
-import org.gradle.api.logging.Logger
-import org.gradle.api.provider.Provider
+import kotlinx.coroutines.runBlocking
 import java.io.File
 import java.util.Locale
 import java.util.concurrent.TimeUnit.MILLISECONDS
 import java.util.concurrent.TimeUnit.MINUTES
-import kotlin.io.path.Path
-import kotlin.io.path.absolutePathString
-import kotlin.io.path.createDirectories
-import kotlin.io.path.createFile
-import kotlin.io.path.exists
-import kotlin.io.path.writeText
 import kotlin.time.Duration.Companion.seconds
 
-class MetricsReporter(
-    private val logger: Logger,
-    private val authToken: Provider<String>,
-    private val buildDir: File,
-) {
-    suspend fun report(
-        report: InMemoryReport,
-        gradleScanId: String?
+
+object InternalA8cCiReporter {
+    private val logger =
+        org.gradle.api.logging.Logging.getLogger(InternalA8cCiReporter::class.java)
```

Comment:
Minor nitpick: can we extract import so we'll have `Logging.getLogger(...)` here only?

Reply:
We already have Ktor Logging import so I used the full package name but we can use an import alias. So I added the type alias here https://github.com/Automattic/measure-builds-gradle-plugin/pull/38/commits/1accd072d3d243f9fab57033780d2ab0d53d3692

Reply:
Oh right, sorry I didn't check it before suggesting the change. The alias looks nice though, thanks 👍